build/ci/pr/Jenkinsfile

```diff
@@ -65,11 +65,14 @@ pipeline {
             withEnv([
                 'REGISTRY=push.docker.elastic.co',
                 'REPOSITORY=eck-snapshots',
-                'IMG_SUFFIX=',
-                'SNAPSHOT_RELEASE=true',
-                'TAG_NAME=${ghprbPullId}'
+                'IMG_NAME=eck-operator',
+                'SNAPSHOT=true'
                 ]) {
-                sh 'make -C build/ci ci-release'
+                sh """
+                    export VERSION=\$(cat $WORKSPACE/operators/VERSION)-${ghprbPullId}
```

Comment:
How this will work with multiple builds within single PR? Should we include date and hash too, for parity with other build versions?

Reply:
Fixed